scripts/target.rb

```diff
@@ -8,7 +8,9 @@ class Target
       'openstack' => ImagefactoryMetadata.new('openstack-kvm', nil, 'qc2'),
       'hyperv'    => ImagefactoryMetadata.new('hyperv', nil, 'vhd'),
       'azure'     => ImagefactoryMetadata.new('hyperv', nil, 'vhd'),
-      'vagrant'   => ImagefactoryMetadata.new('vsphere', 'vagrant-virtualbox', 'box')
+      'vagrant'   => ImagefactoryMetadata.new('vsphere', 'vagrant-virtualbox', 'box'),
+      'libvirt'   => ImagefactoryMetadata.new('openstack-kvm', nil, 'qc2'),
+      'gce'       => ImagefactoryMetadata.new('gce', nil, 'tgz'),
```

Comment:
@geertj Can you change the extension to`.tar.gz`?  That's GCE's requirement and disk can't be created from `.tgz` image.